job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/publishImageGallery.groovy

```diff
@@ -0,0 +1,14 @@
+job('example-1') {
+  publishers {
+    publishImageGallery('some title', './some/path/with/regex/*.png')
```

Comment:
The Image Gallery plugin supports three types of gallery ("Archived images gallery", "In folder comparative archived images gallery" and "Multiple folder comparative archived images gallery"). You are implementing "Archived images gallery", which is fine. But the DSL should leave room for the other gallery types.

Please add another context that allows the other types to be added in the future, e.g.

```groovy
job('example-2') {
    publishers {
        publishImageGalleries {
            archivedImagesGallery('some title', './some/path/with/regex/*.png') {
                imageWidthText(15)
                markBuildAsUnstableIfNoArchivesFound()
            }
        }
    }
}
```